packages/app/src/systems/Settings/pages/ChangePassword/ChangePassword.tsx

```diff
@@ -81,11 +81,12 @@ export function ChangePassword() {
             </Alert>
             <ControlledField
               control={control}
-              name="oldPassword"
+              name="currentPassword"
               label="Current Password"
               render={({ field }) => (
                 <InputPassword
                   {...field}
+                  autoComplete="current-password"
```

Comment:
interesting... does this grab a users saved password from their browser?

Reply:
yes this helps password managers to auto complete